### PR TITLE
New package: SilouhetteStudio.Silouhette version 4.5.770

### LINKED ISSUE #132970: [Package Request]: Silouhette Studio
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://dl.silhcdn.com/1dc240dfb4eb6c5f
Publisher: Silhouette
Package Name: SS_V4.5.770.001_W6R.exe
Description: Silhouette Studio
Package Version: 4.5.770
Installer URL: https://dl.silhcdn.com/1dc240dfb4eb6c5f

### PR DESCRIPTION
**Pull request by Exorcism0666**
### Pull request has been created with [Komac](https://github.com/russellbanks/Komac) v1.11.0 :rocket:

- Resolve: #132970

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132976)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
@Karl-WE If there are any mistakes, don't hesitate to correct them.

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by Exorcism0666** [comment]
image

**Comment by Exorcism0666** [comment]
@Trenly I've added Silent and SilentWithProgress switches and it doesn't work ?? 😭

File: manifests/s/SilouhetteStudio/Silouhette/4.5.770/SilouhetteStudio.Silouhette.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SilouhetteStudio.Silouhette
+PackageVersion: 4.5.770
+InstallerType: exe
+InstallerSwitches:
+  Silent: /exenoui
```
Comment (Trenly) [inline manifests/s/SilouhetteStudio/Silouhette/4.5.770/SilouhetteStudio.Silouhette.installer.yaml:8]:
```suggestion
  Silent: /exenoui /qn
```

File: manifests/s/SilouhetteStudio/Silouhette/4.5.770/SilouhetteStudio.Silouhette.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SilouhetteStudio.Silouhette
+PackageVersion: 4.5.770
+InstallerType: exe
+InstallerSwitches:
+  Silent: /exenoui
+  SilentWithProgress: /exebasicui
```
Comment (Trenly) [inline manifests/s/SilouhetteStudio/Silouhette/4.5.770/SilouhetteStudio.Silouhette.installer.yaml:9]:
```suggestion
  SilentWithProgress: /exenoui /qb
```

File: manifests/s/SilouhetteStudio/Silouhette/4.5.770/SilouhetteStudio.Silouhette.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SilouhetteStudio.Silouhette
+PackageVersion: 4.5.770
+InstallerType: exe
+InstallerSwitches:
+  Silent: /exenoui
+  SilentWithProgress: /exebasicui
+  Custom: ''
```
Comment (Trenly) [inline manifests/s/SilouhetteStudio/Silouhette/4.5.770/SilouhetteStudio.Silouhette.installer.yaml:10]:
```suggestion
```

**Review by Trenly — CHANGES_REQUESTED** [review]
I'm on mobile right now and can’t get to my laptop, but this is an MSI installer inside an exe installer; Based on the help box shown you have to fully suppress the exe UI and then set the right parameter for the MSI install

File: manifests/s/SilouhetteStudio/Silouhette/4.5.770/SilouhetteStudio.Silouhette.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+PackageIdentifier: SilouhetteStudio.Silouhette
+PackageVersion: 4.5.770
+InstallerType: exe
+InstallerSwitches:
+  Silent: /exenoui /qn
+  SilentWithProgress: /exenoui /qb
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```
Comment (OfficialEsco) [inline manifests/s/SilouhetteStudio/Silouhette/4.5.770/SilouhetteStudio.Silouhette.installer.yaml:12]:
```suggestion
- Architecture: x64
```
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20240107031848

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/s/SilouhetteStudio/Silouhette/4.5.770/SilouhetteStudio.Silouhette.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: SilouhetteStudio.Silouhette
+PackageVersion: 4.5.770
+PackageLocale: en-US
+Publisher: Silhouette Studio
```
Comment (OfficialEsco) [inline manifests/s/SilouhetteStudio/Silouhette/4.5.770/SilouhetteStudio.Silouhette.locale.en-US.yaml:7]:
```suggestion
Publisher: Silhouette America
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

Could you add an InstallerUrl?

(Automated response - build 528.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**